src/Sylius/Bundle/PaymentBundle/Resources/config/routing/integrations/sylius.yaml

```diff
@@ -0,0 +1,9 @@
+sylius_webhook_payment_method:
+    path: /payment-method/{code}
```

Comment:
What if I've payment gateway with static single webhook address, but multiple payment methods?

For example I've configured payment methods at sylius as 
> acme_cards
> acme_crypto
> acme_bank_transfer

but at acme merchant panel, I can only configure single webhook address, which will receive notifications for all three payment methods? I've experienced at least 3 payment gateways that do this in recent years.

Maybe it be solved with something like
> /payment-factories/{code}

Reply:
Interesting, this can be an aditional feature for another PR, the payment provider behavior can be replicate to a payment method provider depending only on the current request.